1000mm/step11/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: "Computer Setup"
+step_number: 1
```

Comment:
It looks like this was there before, but it seems strange. Is there a reason why the step number is 1 here but 11 in the permalink?

Reply:
It is supposed to be an 11.